Clicker/ViewControllers/AttendanceViewController.swift

```diff
@@ -137,10 +139,21 @@ class AttendanceViewController: UIViewController {
 
     // MARK: - Action
     @objc func selectAllBtnPressed() {
-        let selectedPollsDateAttendanceArray = pollsDateAttendanceArray.map { PollsDateAttendanceModel(model: $0.model, isSelected: true) }
-        pollsDateAttendanceArray = selectedPollsDateAttendanceArray
-        adapter.performUpdates(animated: false, completion: nil)
-        isExportable = true
+        if !unSelect {
+            let selectedPollsDateAttendanceArray = pollsDateAttendanceArray.map { PollsDateAttendanceModel(model: $0.model, isSelected: true) }
+            pollsDateAttendanceArray = selectedPollsDateAttendanceArray
+            adapter.performUpdates(animated: false, completion: nil)
+            isExportable = true
+            unSelect.toggle()
```

Comment:
if this happens in both the `if else` statement can you place it outside of both blocks? (same with `adapter.performUpdates`)

Reply:
can't you also put `isExportable = !unSelect` outside?